### PR TITLE
[generator][ugc] Skip empty UGCs

### PR DESCRIPTION
**Pull request by milchakov**

### CODE REVIEW

File: generator/generator_tests/ugc_test.cpp [hunk 1]
```diff
@@ -8,16 +8,32 @@
...
+                       INSERT INTO "ratings" VALUES(9826354,'{"osm_id":9826354,"total_rating":-1.0,"based_on":1,"ratings":[{"key":"2","value":3.4},{"key":"3","value":6.0001},{"key":"6","value":0.0001}],"reviews":[{"id":78645323924,"text":{"text":"Изумительно!","lang":"ru"},"author":"Вася","rating":10,"date":1234569}]}');
                        CREATE INDEX key_index ON ratings (key);
                        COMMIT;
 )LLL");
 
+ugc::UGC GetUgcForId(uint32_t id)
+{
+  generator::UGCTranslator tr;
+  tr.CreateDb(g_database);
+  osm::Id osmId = osm::Id(id);
```
Comment (ygorshenin) [inline generator/generator_tests/ugc_test.cpp]:
```c++
osm::Id osmId(id);
```
?

**Comment by ygorshenin** [comment]
Please, rename commit to "Skip empty UGCs".

**Comment by milchakov** [comment]
@ygorshenin PTAL

File: generator/generator_tests/ugc_test.cpp (hunk 1 above)
Comment (milchakov) [reply generator/generator_tests/ugc_test.cpp]: готово

**Pull request merged**